core-web/libs/portlets/dot-ema/src/lib/shared/models.ts

```diff
@@ -38,3 +38,12 @@ export interface ContainerActionPayload {
     contentletID: string;
     personaTag?: string;
 }
+
+export interface NavigationBarItem {
+    icon?: string;
+    iconURL?: string;
+    iconURLActive?: string;
+    label: string;
+    href: string;
+    _active?: boolean;
```

Comment:
why optional?

Reply:
Not needed anymore after the svg refactor